Apps/AdminWebClient/src/ClientApp/src/components/core/Address.vue

```diff
@@ -40,13 +40,12 @@ export default class AddressComponent extends Vue {
 
     private get internationalDestinations(): SelectItem[] {
         // sort destinations alphabetically except place Canada and US at the top
-        const destinations = Object.keys(InternationalDestinations)
-            .filter(
-                (destination) =>
-                    destination !== Countries.CA[0] &&
-                    destination !== Countries.US[0]
-            )
-            .sort();
+        const destinations = Object.keys(InternationalDestinations).filter(
+            (destination) =>
+                destination !== Countries.CA[0] &&
+                destination !== Countries.US[0]
+        );
+        destinations.sort((a, b) => a.localeCompare(b));
```

Comment:
I can't see the critical issues any longer from SonarCloud (I might just not know where to find them). but was it necessary to lose the chaining of the methods?

Reply:
It is because I ran the job against my branch.

Reply:
@BronzBierd I ran it against DEV.  They are back.  https://sonarcloud.io/project/issues?resolved=false&severities=CRITICAL&id=bcgov_healthgateway